database/sql/V067__create_jit_orders.sql

```diff
@@ -0,0 +1,21 @@
+-- Create a new table based on the structure of the existing table
+CREATE TABLE jit_orders AS
+TABLE orders WITH NO DATA;
```

Comment:
I'm not so sure blindly copying the struct of the orders table makes sense for jit orders. I think the table should be more a "jit trade" as each order will come with executed amounts as well (it will be recovered from the data we observe on chain) and fields like `class`, `cancellation_timestamp`, `full_fee_amount`, `settlement_contract` don't make sense in the context of jit orders.

Or is it possible for the same jit order to have multiple settlements?

Reply:
Thanks for the feedback. I am worried that not reusing existing fetching functions will require a serious redesign of existing sql queries for fetching (or, at least, adding separate sql query to fetch jit orders and then merging two vectors of orders). And, afterall, conversion from "nicely defined" table you suggested to regular API format of orders (response of https://api.cow.fi/docs/#/default/get_api_v1_orders__UID_) will have to be implemented somewhere in the code.

Anyway, I will refactor the PR to contain only specific fields for jit orders and we can go from there.

Reply:
I've reconstructed manually all currently active fields on `orders` table and commented out fields that are not related to jit orders or fields that are not recoverable from onchain settlement.